packages/svelte/scripts/process-messages/index.js

```diff
@@ -339,7 +339,7 @@ function transform(name, dest) {
 				};
 
 				for (let i = 0; i < node.expressions.length; i += 1) {
-					const q = node.quasis[i + 1];
+					const q = structuredClone(node.quasis[i + 1]);
```

Comment:
what's this change about?

Reply:
(reverting it doesn't cause any tests to fail)

Reply:
Revert it and rebuild the generated messages then you'll see it - later code does mutate properties and that can pollute the node across codes

Reply:
ah, i see